README.md

```diff
@@ -38,6 +38,10 @@ Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.
 
 Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
 
+## Running with docker
+
+Run `ng build --configuration production` and then `docker-compose up` to startup the docker container with the application
```

Comment:
Not crutial, but you can add additional npm script `"start:docker:compose": "ng build --configuration production && docker-compose up"`.

Reply:
sure I plan to make this nice in this ticket https://backbase.atlassian.net/browse/WF-408 I will make a pr today or tomorrow to do this and other changes